zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -2649,8 +2704,8 @@ components:
           type: string
         operationReference:
           description: |
-            A reference key chosen by the user and will be part of all records resulting
-            from this operation. Must be >0 if provided.
+            A refereA reference key chosen by the user that will be part of all records resulting from this operation.
```

🔧 Something went wrong with this sentence here, I guess.
```suggestion
            A reference key chosen by the user that will be part of all records resulting from this operation.
```